samples-generator/ThirdParty.json

```diff
@@ -0,0 +1,58 @@
+[
+  {
+    "name": "bluebird",
```

Comment:
@lilleyse We shouldn't list devdependencies as part of the thirdparty JSON since we don't use it in the shipped version. According to the package.json in this folder, Bluebird (and other libraries) are dev dependencies, right? Can you please check it in the other folders too?

Reply:
`bluebird` is listed under `dependencies`: https://github.com/CesiumGS/3d-tiles-validator/blob/e4415ccb52595e29c43d575ef29507b343ad85e8/samples-generator/package.json#L25-L36